### ISSUE TITLE
Improve our 404 Page

### ISSUE THREAD

**Issue by joepavitt** [FlowFuse/flowfuse#3772]
### Description

Our in-app 404 page right now is very poor, and relatively easy to come by. With the new "Plain" layout introduced by @cstns, we have an easy win to improve our 404 page.

### Which customers would this be available to

None

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by cstns** [comment]
Current design: 
![image](https://github.com/FlowFuse/flowfuse/assets/20094749/b99b8647-9f82-48d4-8b31-37f430466d34)

**Comment by cstns** [comment]
Will use the plain layout and add a home and back buttons

**Comment by Steve-Mcl** [comment]
Was about to raise this issue. I see this page fairly often and it does zero to inspire me.

Any objections to something a bit more fun / fancy? Nothing crazy, but IMHO, adding a couple of links to back/home is still a bit sterile. There are loads of fun 404 pages (and still professional)

**Comment by joepavitt** [comment]
Yep - crack on @Steve-Mcl

**Comment by cstns** [comment]
@Yndira-E, whenever you have some time, I'd love your input with something flow-fuse themed

**Comment by Steve-Mcl** [comment]
One idea I had was something similar to our excellent home page video animation that draws a Node-RED flow and deploys it.

![image](https://github.com/user-attachments/assets/a98e73cf-43c1-47ad-bda4-42ccd72dcf21)

<br>

For example, an animation that draws an inject -> http request -> debug, deploys, then the mouse presses inject, the debug outputs `404` or "Four oh Four!" or something whimsical.

Of course we can include some static text: some self deprecating reason apologising for the missing page and how we are already busy looking for it (light hearted stuff).

**Comment by Yndira-E** [comment]
How about something like this?
<img width="1439" alt="Screenshot 2024-09-30 at 15 45 56" src="https://github.com/user-attachments/assets/85450551-2489-40a0-9eeb-17f3b879f89d">

Figma file [here](https://www.figma.com/design/rMB2w5ZZUtTC0nxbwLYKJY/FlowForge-Platform?node-id=5323-11444&t=h0SwUHgcXrrrqeXY-1).

**Comment by Yndira-E** [comment]
The only new colour was the light red, but we can remove it if you think it's too much
![404](https://github.com/user-attachments/assets/9fa45fb8-39c4-4fbb-9c15-9fd9d8ed204e)

**Comment by joepavitt** [comment]
Yeah, it's the pastel colouring that made it feel a bit too "cuddly" vs the more vibrant colouring we have elsewhere
